app/services/diagnostics/move_inspector.rb

```diff
@@ -216,9 +236,84 @@ def generate
         else
           @output << "(no person escort record events recorded)\n"
         end
+
+        @output << <<~ENDPER
+
+          YOUTH RISK ASSESSMENT
+          ---------------------
+        ENDPER
+
+        if move.profile&.youth_risk_assessment.present?
+          move.profile.youth_risk_assessment.tap do |yra|
+            @output << "id:\t#{yra.id}\n"
+            @output << "framework version:\t#{yra.framework&.version}\n"
+            @output << "framework_id:\t#{yra.framework_id}\n"
+            @output << "profile_id:\t#{yra.profile_id}\n"
+            @output << "move_id:\t#{yra.move_id}\n"
+            @output << "prefill_source_id:\t#{yra.prefill_source_id}\n"
+            @output << "section_progress:\n"
+            yra.section_progress.each do |section|
+              @output << "* #{section['key']}:\t#{section['status']}\n"
+            end
+            @output << "status:\t#{yra.status}\n"
+            @output << "created at:\t#{yra.created_at}\n"
+            @output << "updated at:\t#{yra.updated_at}\n"
+            @output << "confirmed at:\t#{yra.confirmed_at}\n"
+          end
+        else
+          @output << "(no youth risk assessment recorded)\n"
+        end
+      end
+
+      topics = [move]
+      topics << move.profile.person_escort_record if move&.profile&.person_escort_record.present?
```

Comment:
You can remove the first nil guard on move for this and the following line - this code wouldn't be reached if the move was nil i.e. just use `move.profile&.person_escort_record.present?` Thinking about it don't all moves have a profile too? 🤔

Reply:
now updated... it is possible (though rare) for a move not to have a profile - I think Abeer found a handful on production when checking into another issue a while back.

Reply:
Cool 👍